extensions/wikia/Recirculation/RecirculationController.class.php

```diff
@@ -20,6 +21,7 @@ public function index() {
 		$posts = $fandomDataService->getPosts( $this->type );
 
 		if ( count( $posts ) > 0 ) {
+			$this->response->setCacheValidity( self::CACHE_DURATION );
```

Comment:
Well there are 4 options right now :)

https://github.com/Wikia/app/blob/MAIN-6280/includes/wikia/nirvana/WikiaResponse.class.php#L47

I'll leave it to you - no hard opinions here.

Just did a quick math and
If you estimate 100 hits per minute that means 99,8% of request will be served by Varnish and max stale data for 20min
if you set it to 15 minutes you get 99,9% of request hitting varnish and max stale date of 30min

Good night :)